packages/core/src/app/checkout/Checkout.tsx

```diff
@@ -609,6 +610,8 @@ class Checkout extends Component<
             this.embeddedMessenger.postComplete();
         }
 
+        SubscribeSessionStorage.removeSubscribeStatus();
```

Comment:
I see we only remove the storage value here, which seems enough. 🤔

Reply:
Yes, ideally should only remove it when a payment is successful so just removing it before navigating away to order-confirmation page.

Reply:
I was thinking about the same sessionStorage, but different checkouts. It should be OK.

Reply:
I think should be ok for this instance.